### PR TITLE
Adding ability to ignore actuators

### PR DESCRIPTION
**Pull request by dsanmartim**
Hi Tiago,

I did a quick implementation on this ticket. Can you take a look on the first try and tell me if anything else would be needed?

### CODE REVIEW

File: python/lsst/ts/standardscripts/maintel/m1m3/check_actuators.py [hunk 1]
```diff
@@ -156,6 +163,14 @@ async def configure(self, config):
         self.actuators_to_test = (
             self.m1m3_actuator_ids if config.actuators == "all" else config.actuators
         )
+        if config.ignore_actuators:
+            self.actuators_to_test = list(
+                filter(
+                    lambda actuator: actuator not in config.ignore_actuators,
```
Comment (tribeiro) [inline python/lsst/ts/standardscripts/maintel/m1m3/check_actuators.py:169]: lambda functions are not allowed per our development guidelines: https://developer.lsst.io/python/style.html#lambda-should-not-be-used

I would suggest doing something like:

```
self.actuators_to_test = [actuator_id for actuator_id in self.actuators_to_test if actuator_id not in config.ignore_actuators]
```

It is more clear and expressive than the implementation here.

**Review by tribeiro — CHANGES_REQUESTED** [review]
Please update the version history and take a look at the inline comments I've made. The CI is failing because of the indentation problem I mention on the schema (line 139).

File: python/lsst/ts/standardscripts/maintel/m1m3/check_actuators.py [hunk 2]
```diff
@@ -130,6 +130,13 @@ def get_schema(cls):
                   - type: string
                     enum: ["all"]
                 default: "all"
+            ignore_actuators:
+                description: Actuators to ignore during the bump test.
+                type: array
+                items:
+                    type: number
+                    enum: [{m1m3_actuator_ids_str}]
+            default: []
```
[the remark this answers is not in the document]
Comment (dsanmartim) [reply python/lsst/ts/standardscripts/maintel/m1m3/check_actuators.py]: I saw it just after opening the PR. Already fixed it.

**Review by tribeiro — APPROVED** [review]
Looks good! thanks for the updates! please, squash your commits before merging.

**Comment by dsanmartim** [comment]
Tks for approving it. Merged!

**Pull request merged**